.github/workflows/codeception.yml

```diff
@@ -32,14 +32,14 @@ jobs:
         runs-on: "ubuntu-20.04"
         continue-on-error: ${{ matrix.experimental }}
         env:
-            PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test?serverVersion=${{ matrix.server_version }}"
+            PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test"
```

Comment:
remove `env` from here

Reply:
Done in [26d0cd](https://github.com/pimcore/data-hub/pull/801/commits/26d0cd0f426bc8b5eb7b47c9ad355f5543961da9)